buildmaster/ATLAS_DY_13TEV/metadata.yaml

```
@@ -0,0 +1,59 @@
+setname: "ATLAS_DY_13TEV"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1603.09222"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1436497"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1436497"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "ATLAS"
+
+implemented_observables:
+
+- observable_name: "FID"
+  observable:
+    description: "Measurements of the fiducial and total cross sections and ratios  for W+/-, Z"
+    label: r"$\sigma^{fid}$"
+    units: "[fb]"
+  process_type: "INC"
+  tables: [8, 9, 11]
+  ndata: 3
+  npoints: [1, 1, 1]
+  plotting:
+    kinematics_override: inc_sqrt_scale
+    dataset_label: ""
+    plot_x: y
+    y_label: '$\sigma^{fid}$'
+    figure_by:
+      - boson
+    extra_labels:
+      " ":
+        - $W^-$
+        - $W^+$
+        - Z
+      boson:
+        - W
+        - W
+        - Z
+  kinematic_coverage: [_zero, mu2, sqrt_s]
+  kinematics:
+    variables:
+      _zero: {description: "", label: "", units: ""}
```

```suggestion
      _Zero: {description: "", label: "", units: ""}
```

(maybe I should automatically fill a column with `zero` when one is missing... the constrain of having k1, k2, k3 is silly in both directions...)

---

This would be best indeed! In this case it we don't overcrowd the implementation with spurious variables.

---

I'm not doing this yet! (but I will)

For the `plot_x`, just do whatever was done in the previous plotting file. If things are not working it just means I had not encountered that situation before and I need to fix it.

---

In some of the previous files, `plot_x` are not defined, as is the case for the particular example above for instance https://github.com/NNPDF/nnpdf/blob/master/nnpdfcpp/data/commondata/PLOTTING_ATLAS_WZ_TOT_13TEV.yaml (not sure how exactly it works in such a case).

---

Not even as part of a plotting type file? (if you look at the PLOTTINGTYPE_<PROCESSTYPE> file?)

If also there it is not defined, leave it undefined (same as before applies, the reader won't work but I'll fix it so it behaves as before, maybe there's some magic happening for inclusive cross sections that I've missed)

---

I indeed tried to leave it undefined but somehow the parser complains that it requires that entry.

---

Ok, so for the benefit of producing report comparisons I leave its value to some reasonable kinematic while waiting for the parser to accommodate this. I added a TODO in the description to not forget about this.

---

ok, but don't worry for the comparisons for now, I'll try to fix by! (I hope it's nothing supercomplicated)

---

(let me know when I should look at this again btw)

---

You can look into this now, this is the only ATLAS dataset that not only has one kinematic variable that is zero but also should not have `plot_x` (in the same way as before) as it should plotted as a function of the Gauge bosons.

In [1976617](https://github.com/NNPDF/nnpdf/pull/1866/commits/1976617bdb7b34ff3c6c0d36a07b4775d38a2e40), I have both removed the `_zero` from the kinematics and the `plot_x` in the plotting.

PS: As for the remaining 4 datasets (inc the JETS), I will finish them by early next week.